plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -210,7 +217,22 @@ class SentryPlugin : Plugin<Project> {
                     // if its a bundle aab, assemble might not be executed, so we hook into bundle task
                     bundleTask?.configure { it.finalizedBy(uploadSentryNativeSymbolsTask) }
                 } else {
-                    project.logger.info("[sentry] uploadSentryNativeSymbols won't be executed")
+                    project.logger.info { "uploadSentryNativeSymbols won't be executed" }
+                }
+
+                project.afterEvaluate {
```

Comment:
Is this necessary? If yes, please add a comment on why.

Reply:
yep, the dependency configuration can only be resolved after the configuration/evaluation phase is done. Ideally this would've been a task, but I had a hard time finding out which task I should depend on - seems like AGP starts its transforms without binding to a specific task based on my tests (and we need to know the SDK version before the transforms are executed). Will leave a comment 👍